### ISSUE TITLE
Migrate spec import to new location docs/spec

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp.dev#5821]
Once https://github.com/ampproject/amphtml/pull/34160 has landed. We probably should import all docs from master to avoid syncing the change with the next amp release.

**Comment by patrickkettner** [comment]
@sebastianbenz just to be sure -  by `master` do you mean `main`?

**Comment by sebastianbenz** [comment]
Yes, but the parameter is still named master in our code. BTW, we're already defaulting to main for the spec import, so we should be fine: https://github.com/ampproject/amp.dev/blob/future/platform/lib/pipeline/specImporter.js#L51
